extensions/skyportal/skyportal/handlers/api/alert.py

```diff
@@ -226,26 +223,134 @@ async def get(self, objectId: str = None):
                         },
                     ],
                 },
-                # "kwargs": {
-                #     "max_time_ms": 10000
-                # }
+                "kwargs": {"max_time_ms": 10000},
             }
 
             candid = self.get_query_argument("candid", None)
             if candid:
                 query["query"]["pipeline"][0]["$match"]["candid"] = int(candid)
 
-            resp = self.query_kowalski(query=query)
+            response = kowalski.query(query=query)
 
-            if resp.status_code == requests.codes.ok:
-                alert_data = bj.loads(resp.text).get("data")
+            if response.get("status", "error") == "success":
+                alert_data = response.get("data")
                 return self.success(data=alert_data)
             else:
-                return self.error(f"Failed to fetch data for {objectId} from Kowalski")
+                return self.error(f"Failed to fetch data for {object_id} from Kowalski")
 
-        except Exception:
-            _err = traceback.format_exc()
-            return self.error(f"failure: {_err}")
+        # executing a general search
+        object_id = self.get_query_argument("objectId", None)
```

Comment:
use different variable name than above

Reply:
It's now `object_ids` since multiple comma-separated ids are now allowed.